src/pages/ClusterPage/new.js

```diff
@@ -207,6 +207,12 @@ export const machineMarks = {
 	},
 };
 
+const validOpenFaasPlans = [
+	'2019-production-4',
+	'2019-production-3',
+	'2019-production-2',
```

Comment:
@lakhansamani Shouldn't it be also available for enterprise plans?

Reply:
this for clusters only, and currently we are not supporting this feature in BYOC clusters, because of system limitations.